samples/nrf9160/azure_iot_hub/README.rst

```diff
@@ -77,7 +77,7 @@ If DPS is used, configure the following library options:
    The sample sets the option :kconfig:`CONFIG_MQTT_KEEPALIVE` to the maximum allowed value, 1767 seconds (29.45 minutes) as specified by Azure IoT Hub.
    This is to limit the IP traffic between the device and the Azure IoT Hub message broker for supporting a low power sample.
    However, note that in certain LTE networks, the NAT timeout can be considerably lower than 1767 seconds.
-   So as a recommendation, and to prevent the likelihood of getting disconnected unexpectedly, the option :kconfig:`CONFIG_MQTT_KEEPALIVE` must be set to the lowest of the aforementioned timeout limits (Maximum allowed MQTT keepalive and NAT timeout).
+   So as a recommendation, and to prevent the likelihood of getting disconnected unexpectedly, set the option :kconfig:`CONFIG_MQTT_KEEPALIVE` to the lowest of the aforementioned timeout limits (Maximum allowed MQTT keepalive and NAT timeout).
```

Comment:
```suggestion
   As a recommendation, and to prevent the likelihood of getting disconnected unexpectedly, set the option :kconfig:`CONFIG_MQTT_KEEPALIVE` to the lowest timeout limit (Maximum allowed MQTT keepalive and NAT timeout).
```